protocol-units/bridge/shared/src/types.rs

```diff
@@ -36,6 +36,12 @@ where
 	}
 }
 
+impl Into<BridgeTransferId> for u64 {
```

Comment:
The idiomatic way in Rust is to implement `From` rather than `Into`. The latter is then available via the blanket implementation.